vulkan_helpers/helper_functions.cpp

```diff
@@ -1313,6 +1329,21 @@ VkPipelineCache CreateDefaultPipelineCache(VkDevice* device) {
   return VkPipelineCache(cache, nullptr, device);
 }
 
+// Writes the given pipeline cache to the given location on disk.
+void WritePipelineCache(VkDevice* device, VkPipelineCache* cache, const char* location) {
+   device->GetLogger()->LogInfo("Wrote pipeline cache to \"", location, "\"");
```

Comment:
nit: indentation is 1 space too many I think.

Reply:
Also, should this be after the actual write, rather than before?

Reply:
Probably should  be after the write